.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -152,7 +171,7 @@ stages:
 
       # upload a loadtest config file from previous task
       - task: AzureCLI@2
-        displayName: 'Upload auto-generated load test config file for Azure Load Test'
+        displayName: 'Upload auto-generated config file'
```

Comment:
why auto-generated?

Reply:
it was autogenerated in an earlier version but currently not used anymore - dropping the whole task now